temporal-sdk/src/main/java/io/temporal/internal/client/ScheduleProtoUtil.java

```diff
@@ -128,6 +130,11 @@ public ScheduleAction actionToProto(io.temporal.client.schedules.ScheduleAction
         workflowRequest.setInput(inputArgs.get());
       }
 
+      RetryOptions retryOptions = startWorkflowAction.getOptions().getRetryOptions();
```

Comment:
```suggestion
      RetryOptions retryOptions = wfOptions.getRetryOptions();
```

(pedantic, can ignore)